packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/capabilities/ProjectCapabilitiesControllerTests.java

```diff
@@ -60,6 +101,18 @@ public class ProjectCapabilitiesControllerTests extends AbstractIntegrationTests
     @Autowired
     private IProjectRepository projectRepository;
 
+    @Autowired
+    private RenameProjectMutationRunner renameProjectMutationRunner;
+
+    @Autowired
+    private EditingContextEventSubscriptionRunner editingContextEventSubscriptionRunner;
+
+    @Autowired
+    private InvokeEditingContextActionMutationRunner invokeEditingContextActionMutationRunner;
+
+    @Autowired
+    private LibrariesQueryRunner librariesQueryRunner;
```

Comment:
You are testing way too much different stuff in this test class, now inside of the "project capabilities controller tests", you are even testing stuff related to libraries. This class should be split in various smaller classes.

Reply:
Already tested in `LibraryControllerWithDemoProfileIntegrationTests` introduced in #5334